### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by flakey5**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1 +1,10 @@
 # Documentation
```
Comment (mcollina) [inline docs/README.md]: I would put this in the root of the repo

File: docs/dev-setup.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Development Setup
```
Comment (mcollina) [inline docs/dev-setup.md]: This should be CONTRIBUTING.md

**Comment by iamfortune** [comment]
This looks good to me, @mcollina. I believe we will update it as we add more features before release. Can I merge?

**Review by iamfortune — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,12 @@
 # ai-warp
+
 Platformatic Stackable to interact with AI services
+
```
Comment (mcollina) [inline README.md:28]: Can you add some docs on how to use this in the readme?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,37 @@
...
+To get started using AI Warp, all you need to do is three steps:
+
+ 1. Generate an AI Warp app. As of now, this can only be done through this repository.
+
+    ```bash
+    npm install
+    npm run build
+    node ./dist/cli/create.js
+    ```
```
Comment (mcollina) [inline README.md:15]: Ok for now, I'llo change it prior to release

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**